File: led.c
// Including Library
#include <stdio.h>
#include <wiringPi.h>

// Define Using PinNo and Other Variables
#define LED_PIN 4   //GPIO 23

int main (void)
{
    // Initializing GPIO Port
    if (wiringPiSetup() == -1)
        return 1;
    // Initializing Pins
    pinMode(LED_PIN, OUTPUT);

    // looping program
    for (;;)
    {
        // On
        digitalWrite (LED_PIN, HIGH);
        // Sleep
        delay (1000);
        // Off
        digitalWrite (LED_PIN, LOW); 
        // Sleep
        delay (1000);
    }
    return 0;
}

File: led.py
# Including Library
import RPi.GPIO as GPIO
import time

# Define Using PinNo and Other Variables
LED_PIN = 23

# Initializing GPIO Port
GPIO.setmode(GPIO.BCM)
# Initializing Pins
GPIO.setup(LED_PIN, GPIO.OUT)

# looping program
while(True):
	# On
    GPIO.output(LED_PIN, True)
    # Sleep
    time.sleep(1)
    # Off
    GPIO.output(LED_PIN, False)
    # Sleep
    time.sleep(1)
